source/agora/test/Byzantine.d

```diff
@@ -230,10 +230,10 @@ private class ByzantineManager (bool addSpyValidator = false,
     }
 }
 
-/// Block should be added if we have all 3 validators signing (our SpyValidator does not sign)
+/// Block should be added if we have all 6 validators signing (our SpyValidator does not sign)
 unittest
 {
-    TestConf conf = { validators : 4, quorum_threshold : 66 };
+    TestConf conf = { quorum_threshold : 66 };
```

Comment:
The thresholds should have been updated. The idea is to set the threshold to N - 1 if one of the nodes in the test does not sign. 66% of 6 nodes is 4 nodes (rounded up). We need a value that matches 5/6 nodes.

Reply:
I will create a new PR for this to be corrected and also remove `validators` as a configuration option.